src/index.js

```diff
@@ -259,7 +313,9 @@ window.onload = async (event) => {
 
     // Add text to nodes with border
     nodes.append('text')
-        .text((d) => d.data.type)
+        .text((d) => {
+            return `${d.data.type} - ${d.data.content && d.data.content.body} (depth=${d.data.depth})`;
```

Comment:
Updated DAG visualization with depth, ![](https://user-images.githubusercontent.com/558581/106591475-f094ce00-6513-11eb-8022-d9df1a229edc.png)

See https://github.com/matrix-org/complement/pull/68#discussion_r568507353 for full context